exercises/chapter3/test/no-peeking/Solutions.purs

```diff
@@ -12,6 +12,10 @@ findEntryByStreet streetName = filter filterEntry >>> head
   filterEntry :: Entry -> Boolean
   filterEntry e = e.address.street == streetName
 
+-- Example alternative implementation using property accessor and composition
+findEntryByStreet' :: String -> AddressBook -> Maybe Entry
+findEntryByStreet' streetName = filter (_.address.street >>> ((==) streetName)) >>> head
```

Comment:
```suggestion
findEntryByStreet' streetName = filter (_.address.street >>> eq streetName) >>> head
```

Reply:
Haha. `(==)` versus `eq` was exactly why I went down the rabbit hole of introducing the use of parens to turn `==` into a prefix function.